LongestPalindrome.py

```diff
@@ -0,0 +1,16 @@
+#Time complexity: O(n)
+#Space complexity: O(n)
```

Comment:
Space will be O(1) at max all the valid ascii characters can be there in map i.e. 256 which is constant